examples/water/se_atten_non_compressible/input.json

```diff
@@ -0,0 +1,87 @@
+{
+  "_comment": " model parameters",
+  "model": {
+    "type_map": [
+      "O",
+      "H"
+    ],
+    "descriptor": {
+      "type": "se_atten",
+      "stripped_type_embedding": false,
```

Comment:
I think @wanghan-iapcm means setting this option to true with 2 attention layers.